docs/widgets.md

```diff
@@ -433,3 +441,39 @@ Example := Window {
     }
 }
 ```
+
+## Selecting a Widget Style
+
+The widget style is selected at compile time of your project. The details depend on which programming
+language you're using Slint with.
+
+<details data-snippet-language="rust">
+<summary>Selecting a Widget Style when using Slint with Rust:</summary>
+
+Before you start your compilation, you can select the style by setting the `SLINT_STYLE` variable
```

Comment:
the SLINT_STYLE works with everything: the viewer, the lsp, the rust macro, ...

Reply:
For the macro I think these docs are correct, aren't they?

But yes, for the viewer and the lsp we also need sections then. Let me see about adding some in a separate PR.